library/posture.c

```diff
@@ -254,8 +289,9 @@ void ziti_send_posture_data(ziti_context ztx) {
         if (ztx->opts->pq_domain_cb != NULL) {
             ztx->opts->pq_domain_cb(ztx, domainInfo->query->id, ziti_pr_handle_domain);
         } else {
-            ZITI_LOG(DEBUG, "%s cb not set requested for: service %s, policy: %s, check: %s", PC_DOMAIN_TYPE,
+            ZITI_LOG(VERBOSE, "using default %s cb for: service %s, policy: %s, check: %s", PC_DOMAIN_TYPE,
```

Comment:
why this one to verbose when the others are left at debug?

Reply:
i'll change all to verbose